blobit-core/src/main/java/org/blobit/core/api/BucketConfiguration.java

```diff
@@ -28,7 +28,8 @@
 
     public static final BucketConfiguration DEFAULT = new BucketConfiguration();
 
-    private int replicaCount = 1;
+    private int replicaCount = Configuration.REPLICATION_FACTOR_DEFAULT;
+    private long inactivityTime = Configuration.LEADER_INACTIVITY_TIME_DEFAULT;
```

Comment:
can't we add some more generic "metadataStoreProperties" ?
this inactivityTime is very generic

thinking about the future....HerdDB is the current only MetadataStore, as you know we could go with a generic transactional JDBC resource